### PR TITLE
New release

### PR DESCRIPTION
**Pull request by loicgasser**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -214,7 +214,7 @@ rancherdeployprod: guard-RANCHER_ACCESS_KEY \
                   guard-RANCHER_SECRET_KEY \
                   guard-RANCHER_URL
 	export RANCHER_DEPLOY=true && make composetemplateprod
-	$(call start_service,$(RANCHER_ACCESS_KEY),$(RANCHER_SECRET_KEY),$(RANCHER_URL),prod)
+	$(call start_service,$(RANCHER_ACCESS_KEY_PROD),$(RANCHER_SECRET_KEY_PROD),$(RANCHER_URL_PROD),prod)
```
Comment (procrastinatio) [inline Makefile]: I dont't like the idea to change variable names. But if you do, also change the `guard` values

**Review by procrastinatio — CHANGES_REQUESTED** [review]
some changes

File: Makefile (hunk 1 above)
Comment (loicgasser) [reply Makefile]: It's a different server, that is why.
`RANCHER_ACCESS_KEY` , `RANCHER_SECRET_KEY`, `RANCHER_URL_PROD`
are the default variables for rancher.
I don't think we want to have variable that deploy to prod per default. I did the same in proxywms.

**Comment by loicgasser** [comment]
comment addressed.

**Review by procrastinatio — APPROVED** [review]
OK for me

**Comment by loicgasser** [comment]
thanks I deployed to dev and Andrea will change to the backend_checker again

**Comment by loicgasser** [comment]
there is an issue

**Comment by loicgasser** [comment]
I am coming with another PR, for the logging.

**Pull request merged**